<?xml version="1.0" encoding="utf-8"?>
<TestRecord xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
  <Results>
    <UnitTestResult>
      <TestDate>2013-10-10T13:36:57</TestDate>
      <Status />
      <Passed>1</Passed>
      <Errors>0</Errors>
      <Failures>0</Failures>
      <Inconclusive>0</Inconclusive>
      <NotRunnable>0</NotRunnable>
      <Skipped>0</Skipped>
      <Ignored>0</Ignored>
      <Time />
      <Message>Test erfolgreich

Ausf√ºhrungszeit: 0.04ms</Message>
    </UnitTestResult>
    <UnitTestResult>
      <TestDate>2013-10-10T13:37:29</TestDate>
      <Status />
      <Passed>1</Passed>
      <Errors>0</Errors>
      <Failures>1</Failures>
      <Inconclusive>0</Inconclusive>
      <NotRunnable>0</NotRunnable>
      <Skipped>0</Skipped>
      <Ignored>0</Ignored>
      <Time />
      <Message>One or more child tests had errors</Message>
    </UnitTestResult>
  </Results>
  <Tests>
    <TestRecord Name="FindBack">
      <Results>
        <UnitTestResult>
          <TestDate>2013-10-10T13:36:57</TestDate>
          <Status />
          <Passed>1</Passed>
          <Errors>0</Errors>
          <Failures>0</Failures>
          <Inconclusive>0</Inconclusive>
          <NotRunnable>0</NotRunnable>
          <Skipped>0</Skipped>
          <Ignored>0</Ignored>
          <Time />
          <Message>Test successful

Execution time: 0.03ms</Message>
        </UnitTestResult>
        <UnitTestResult>
          <TestDate>2013-10-10T13:37:29</TestDate>
          <Status />
          <Passed>1</Passed>
          <Errors>0</Errors>
          <Failures>1</Failures>
          <Inconclusive>0</Inconclusive>
          <NotRunnable>0</NotRunnable>
          <Skipped>0</Skipped>
          <Ignored>0</Ignored>
          <Time />
          <Message>One or more child tests had errors</Message>
        </UnitTestResult>
      </Results>
      <Tests>
        <TestRecord Name="Core">
          <Results>
            <UnitTestResult>
              <TestDate>2013-10-10T13:36:57</TestDate>
              <Status />
              <Passed>1</Passed>
              <Errors>0</Errors>
              <Failures>0</Failures>
              <Inconclusive>0</Inconclusive>
              <NotRunnable>0</NotRunnable>
              <Skipped>0</Skipped>
              <Ignored>0</Ignored>
              <Time />
              <Message>Test successful

Execution time: 0.03ms</Message>
            </UnitTestResult>
            <UnitTestResult>
              <TestDate>2013-10-10T13:37:29</TestDate>
              <Status />
              <Passed>1</Passed>
              <Errors>0</Errors>
              <Failures>1</Failures>
              <Inconclusive>0</Inconclusive>
              <NotRunnable>0</NotRunnable>
              <Skipped>0</Skipped>
              <Ignored>0</Ignored>
              <Time />
              <Message>One or more child tests had errors</Message>
            </UnitTestResult>
          </Results>
          <Tests>
            <TestRecord Name="Test">
              <Results>
                <UnitTestResult>
                  <TestDate>2013-10-10T13:36:57</TestDate>
                  <Status />
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>0</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>Test successful

Execution time: 0.03ms</Message>
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2013-10-10T13:37:29</TestDate>
                  <Status />
                  <Passed>1</Passed>
                  <Errors>0</Errors>
                  <Failures>1</Failures>
                  <Inconclusive>0</Inconclusive>
                  <NotRunnable>0</NotRunnable>
                  <Skipped>0</Skipped>
                  <Ignored>0</Ignored>
                  <Time />
                  <Message>One or more child tests had errors</Message>
                </UnitTestResult>
              </Results>
              <Tests>
                <TestRecord Name="TestClass">
                  <Results>
                    <UnitTestResult>
                      <TestDate>2013-10-10T13:36:57</TestDate>
                      <Status />
                      <Passed>1</Passed>
                      <Errors>0</Errors>
                      <Failures>0</Failures>
                      <Inconclusive>0</Inconclusive>
                      <NotRunnable>0</NotRunnable>
                      <Skipped>0</Skipped>
                      <Ignored>0</Ignored>
                      <Time />
                      <Message>Test successful

Execution time: 0.03ms</Message>
                    </UnitTestResult>
                    <UnitTestResult>
                      <TestDate>2013-10-10T13:37:29</TestDate>
                      <Status />
                      <Passed>1</Passed>
                      <Errors>0</Errors>
                      <Failures>1</Failures>
                      <Inconclusive>0</Inconclusive>
                      <NotRunnable>0</NotRunnable>
                      <Skipped>0</Skipped>
                      <Ignored>0</Ignored>
                      <Time />
                      <Message>One or more child tests had errors</Message>
                    </UnitTestResult>
                  </Results>
                  <Tests>
                    <TestRecord Name="When2_THenIsOk">
                      <Results>
                        <UnitTestResult>
                          <TestDate>2013-10-10T13:36:57</TestDate>
                          <Status />
                          <Passed>1</Passed>
                          <Errors>0</Errors>
                          <Failures>0</Failures>
                          <Inconclusive>0</Inconclusive>
                          <NotRunnable>0</NotRunnable>
                          <Skipped>0</Skipped>
                          <Ignored>0</Ignored>
                          <Time />
                          <Message>Test successful

Execution time: 0.02ms</Message>
                        </UnitTestResult>
                        <UnitTestResult>
                          <TestDate>2013-10-10T13:37:29</TestDate>
                          <Status />
                          <Passed>1</Passed>
                          <Errors>0</Errors>
                          <Failures>0</Failures>
                          <Inconclusive>0</Inconclusive>
                          <NotRunnable>0</NotRunnable>
                          <Skipped>0</Skipped>
                          <Ignored>0</Ignored>
                          <Time />
                          <Message>Test successful

Execution time: 0.00ms</Message>
                        </UnitTestResult>
                      </Results>
                      <Tests>
                        <TestRecord Name="When2_THenIsOk()">
                          <Results>
                            <UnitTestResult>
                              <TestDate>2013-10-10T13:36:57</TestDate>
                              <Status />
                              <Passed>1</Passed>
                              <Errors>0</Errors>
                              <Failures>0</Failures>
                              <Inconclusive>0</Inconclusive>
                              <NotRunnable>0</NotRunnable>
                              <Skipped>0</Skipped>
                              <Ignored>0</Ignored>
                              <Time />
                              <Message>Test successful

Execution time: 0.02ms</Message>
                              <ConsoleOutput />
                              <ConsoleError />
                            </UnitTestResult>
                            <UnitTestResult>
                              <TestDate>2013-10-10T13:37:29</TestDate>
                              <Status />
                              <Passed>1</Passed>
                              <Errors>0</Errors>
                              <Failures>0</Failures>
                              <Inconclusive>0</Inconclusive>
                              <NotRunnable>0</NotRunnable>
                              <Skipped>0</Skipped>
                              <Ignored>0</Ignored>
                              <Time />
                              <Message>Test successful

Execution time: 0.00ms</Message>
                              <ConsoleOutput />
                              <ConsoleError />
                            </UnitTestResult>
                          </Results>
                        </TestRecord>
                      </Tests>
                    </TestRecord>
                    <TestRecord Name="When2_THenIsNotOk">
                      <Results>
                        <UnitTestResult>
                          <TestDate>2013-10-10T13:37:29</TestDate>
                          <Status />
                          <Passed>0</Passed>
                          <Errors>0</Errors>
                          <Failures>1</Failures>
                          <Inconclusive>0</Inconclusive>
                          <NotRunnable>0</NotRunnable>
                          <Skipped>0</Skipped>
                          <Ignored>0</Ignored>
                          <Time />
                          <Message>One or more child tests had errors</Message>
                        </UnitTestResult>
                      </Results>
                      <Tests>
                        <TestRecord Name="When2_THenIsNotOk()">
                          <Results>
                            <UnitTestResult>
                              <TestDate>2013-10-10T13:37:29</TestDate>
                              <Status />
                              <Passed>0</Passed>
                              <Errors>0</Errors>
                              <Failures>1</Failures>
                              <Inconclusive>0</Inconclusive>
                              <NotRunnable>0</NotRunnable>
                              <Skipped>0</Skipped>
                              <Ignored>0</Ignored>
                              <Time />
                              <Message>Expected 3, but found 2.</Message>
                              <StackTrace>   at FluentAssertions.Execution.LateBoundTestFramework.Throw(String message) in d:\Workspace\Github\FluentAssertions\FluentAssertions.Net35\Execution\LateBoundTestFramework.cs:line 25
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message) in d:\Workspace\Github\FluentAssertions\FluentAssertions.Net35\Execution\DefaultAssertionStrategy.cs:line 23
   at FluentAssertions.Execution.AssertionScope.FailWith(String failureMessage, Object[] failureArgs) in d:\Workspace\Github\FluentAssertions\FluentAssertions.Net35\Execution\AssertionScope.cs:line 165
   at FluentAssertions.Numeric.NumericAssertions`1.Be(T expected, String reason, Object[] reasonArgs) in d:\Workspace\Github\FluentAssertions\FluentAssertions.Net35\Numeric\NumericAssertions.cs:line 44
   at FindBack.Core.Test.TestClass.When2_THenIsNotOk() in c:\dev\VSProjects\FindBack\Dev\source\FindBack\FindBack.Core.Test\TestClass.cs:line 20</StackTrace>
                              <ConsoleOutput />
                              <ConsoleError />
                            </UnitTestResult>
                          </Results>
                        </TestRecord>
                      </Tests>
                    </TestRecord>
                  </Tests>
                </TestRecord>
              </Tests>
            </TestRecord>
          </Tests>
        </TestRecord>
      </Tests>
    </TestRecord>
  </Tests>
</TestRecord>